ens/utils.py

```diff
@@ -23,38 +39,46 @@
 default = object()
 
 
-def Web3():
-    from web3 import Web3
-    return Web3
+if TYPE_CHECKING:
+    from web3 import Web3 as Web3Type  # noqa: F401
```

Comment:
I'm  not  a fan of using `Web3Type` here, but it was the simplest way I could think of since `Web3` is defined on line 49.

Reply:
Is the issue just with the name?  You can do `import Web3 as _Web3` maybe?